src/app/modules/organisations/organisation-edit/organisation-edit.component.ts

```diff
@@ -38,23 +38,23 @@ export class OrganisationEditComponent implements OnInit {
   getOrganisationDetails() {
     this.organisationService.organisationDetails(this.organisationId).subscribe(data => {
       this.editOrganisationDetails = data['result'];
-      console.log(' this.editOrganisationDetails', this.editOrganisationDetails);
-
     }, error => {
-      this.commonServiceService.commonSnackBar(error.error.message.params.errmsg, 'Dismiss', 'top', 1000)
+      this.commonServiceService.errorHandling(error);
```

Comment:
Can we do the error handling in one commonplace?

Reply:
we can handle but due to files download we cannot do right now